### DISCUSSION TITLE
can I use a reserved IP and DHCP rather then a static IP?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#9374] [Unsupported Versions]
I use spectrum as my cable provider. Spectrum can reserve you IPv4 address. They can only make a static IP for businesses not for residential customers. This means it will not dynamically change this IPv4 address to work like a static IP? Neither of these will not change at any time, so when setting up security onion, I only need to run  DHCP instead of static. The idea for static is an IP address that will not change.

**Comment by dougburks** [comment]
If you use a static IP address from your Internet provider, that would be a public IP address on the Internet. You most likely do not want to expose your Security Onion management interface to the Internet. Instead, simply use an IP address in your LAN.

**Reply by iqworks** [reply]
Thanks Doug, are you talking about using one of these?
  
![image](https://user-images.githubusercontent.com/5569263/207615788-a7ff64ee-f4f1-45a9-9fdd-5b8e2268754f.png)

**Reply by dougburks** [reply]
Yes, if you're using VMware, then you should use the IP address ranges built into VMware.

We've discussed many times previously that basic networking knowledge is a prerequisite for cybersecurity in general and that basic networking questions are beyond the scope of this discussions forum:
https://github.com/Security-Onion-Solutions/securityonion/discussions/5131
https://github.com/Security-Onion-Solutions/securityonion/discussions/5134
https://github.com/Security-Onion-Solutions/securityonion/discussions/5652
https://github.com/Security-Onion-Solutions/securityonion/discussions/8519
